tox.ini

```diff
@@ -3,13 +3,15 @@ envlist =
     {py27,py33,py34,py35}-django18
     {py27,py34,py35}-django19
     {py27,py34,py35}-django110
+    {py27,py34,py35,py36}-django111
 
 [testenv]
 setenv =
     PYTHONPATH = {toxinidir}:{toxinidir}/mozilla_django_oidc
 commands = python runtests.py
 deps =
-    django-18: Django>=1.8,<1.9
-    django-19: Django>=1.9,<1.10
-    django-110: Django>=1.10
+    django18: Django>=1.8,<1.9
+    django19: Django>=1.9,<1.10
+    django110: Django>=1.10,<1.11
+    django111: Django>=1.11
```

Comment:
These were specified wrong originally because these bits don't have a hyphen. I think tox was ignoring that and just keeping whatever the `requirements.txt` file installed. Thus tox was just testing the same Django version over and over again.